ql/src/java/org/apache/hadoop/hive/ql/exec/vector/ptf/VectorPTFOperator.java

```diff
@@ -318,6 +318,7 @@ protected void initializeOp(Configuration hconf) throws HiveException {
     currentPartitionIntervalDayTimes = new HiveIntervalDayTime[partitionKeyCount];
 
     VectorExpression.doTransientInit(partitionExpressions, hconf);
+    VectorExpression.doTransientInit(orderExpressions, hconf);
```

Comment:
could you please move doTransientInit calls to the beginning of the method, after super.initializeOp?
I don't think it will change behavior but I can see every vectorized operator that calls this method, calls at the beginning of initializeOp, I would try to stay aligned with the pattern :)

Reply:
Sure